### PR TITLE
Added events free for all

### PR DESCRIPTION
**Pull request by tommygonzaleza**

### CODE REVIEW

**Comment by tommygonzaleza** [comment]
https://github.com/breatheco-de/breatheco-de/issues/6461

File: breathecode/events/permissions/consumers.py [hunk 1]
```diff
@@ -61,7 +62,8 @@ def event_by_url_param(context: PermissionContextType, args: tuple, kwargs: dict
         | Q(cohort__available_as_saas=None, cohort__academy__available_as_saas=False),
         user=request.user).exists()
 
-    if not is_host and (not is_free_for_bootcamps or not user_with_available_as_saas_false):
+    if not is_host and not is_free_for_all and (not is_free_for_bootcamps
```
Comment (jefer94) [inline breathecode/events/permissions/consumers.py:65]: Add validation in the model, if it's free for all it's free for bootcamp too, set in these cases `free_for_bootcamps` to `None`, then apply an or condition `(event.free_for_all or event.free_for_bootcamps)`

**Review by jefer94 — CHANGES_REQUESTED** [review]

File: breathecode/events/models.py [hunk 2]
```diff
@@ -182,6 +182,8 @@ def __init__(self, *args, **kwargs):
                             validators=[validate_language_code])
     currency = models.CharField(max_length=3, choices=CURRENCIES, default=USD, blank=True)
     tags = models.CharField(max_length=100, default='', blank=True)
+    free_for_all = models.BooleanField(
```
Comment (jefer94) [inline breathecode/events/models.py:185]: There have so many undefined variables
https://github.com/breatheco-de/apiv2/actions/runs/6882340259/job/18720662409?pr=1163

**Review by jefer94 — CHANGES_REQUESTED** [review]

**Comment by jefer94** [comment]
The code is bad formatted

File: breathecode/events/models.py (hunk 2 above)
Comment (tommygonzaleza) [reply breathecode/events/models.py:185]: The undefined variables are not related to the code I added. Maybe they should be fixed in a different issue

Comment (tommygonzaleza) [reply breathecode/events/models.py:185]: It's solved now, you were right

**Pull request merged**